docs/source/examples/workspaces/openshift/PinFile

```diff
@@ -6,34 +6,40 @@ openshift-new:
       - resource_group_name: test1
         resource_group_type: openshift
         resource_definitions:
-          - name: openshift
+          - name: hellopod
```

Comment:
is this how it is in upshift? can we change it to more generic and ocp related name?

Reply:
@skatlapa : its just the name of the pod . So it can be anything .  However if using the template definition it takes the meta name.